repairnator/src/main/java/fr/inria/spirals/repairnator/process/step/CheckoutSourceCodeForPreviousBuild.java

```diff
@@ -0,0 +1,47 @@
+package fr.inria.spirals.repairnator.process.step;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.api.errors.GitAPIException;
+
+import fr.inria.spirals.repairnator.process.ProjectState;
+import fr.inria.spirals.repairnator.process.inspectors.ProjectInspector;
+
+/**
+ * Created by fermadeiral.
+ */
+public class CheckoutSourceCodeForPreviousBuild extends CloneRepository {
+
```

Comment:
The thing is the `CheckoutPreviousBuild` class is to checkout the whole build, and `CheckoutSourceCodeForPreviousBuild` class is to checkout only the source code of the previous build (Bn-1) to test it with the test cases in the current build (Bn).